clients/privacy-center/pages/index.tsx

```diff
@@ -89,6 +91,11 @@ const Home: NextPage = () => {
         onOpen={onConsentModalOpen}
       />
     );
+    if (router.query?.showConsentModal === "true") {
+      // manually override whether to show the consent modal given
+      // the query param `showConsentModal`
+      isConsentModalOpen = true;
```

Comment:
I think you maybe able to call the `onConsentModalOpen` function here instead and it'll update `isConsentModalOpenConst` to true

Reply:
Thanks! I'll give it a go

Reply:
Ah great, thanks!